src/widgetastic_patternfly4/dropdown.py

```diff
@@ -187,6 +187,11 @@ def item_select(self, item, handle_alert=None, **kwargs):
                 self.logger.warning("There is an unexpected alert present.")
                 pass
 
+    @property
+    def button_text(self):
+        """Returns a string of the selected item."""
```

Comment:
There is no such thing as selected item in Dropdown. Here you return the name of the dropdown.

Reply:
Please write a correct docstring here or remove tis property. I don't see any usage of it.

Reply:
Corrected, the usage is in IQE, not here. https://gitlab.cee.redhat.com/insights-qe/advisor/merge_requests/106